VMEncryption/main/Common.py

```diff
@@ -20,7 +20,7 @@
 class CommonVariables:
     utils_path_name = 'Utils'
     extension_name = 'AzureDiskEncryptionForLinuxTest'
```

Comment:
Remove the Test at the end of the extension_name

Reply:
This is done for test publish till we fix our EDP pipeline.